README.md

```diff
@@ -1,22 +1,22 @@
 [![Build](https://github.com/etn-ccis/blui-react-cli-templates/actions/workflows/blui-ci.yml/badge.svg?branch=master)](https://github.com/etn-ccis/blui-react-cli-templates/actions/workflows/blui-ci.yml)
 
-# React CLI Templates
+# React + TypeScript + Vite Templates
 
-These are project templates that are used by Create React App when starting a new Brightlayer UI project. They are used behind the scenes by the Brightlayer UI CLI.
+These are project templates that are used for starting a new Brightlayer UI project. The templates are configured with the basic Brightlayer UI libraries for themes and components and configures the application to use them.
 
 ## Available Templates
 
 ### Blank
 
-The [Blank template](https://www.npmjs.com/package/@brightlayer-ui/cra-template-blank-typescript) provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use.
+The [Blank template](./templates/blank-typescript/README.md) provides a bare-bones application with Brightlayer UI components and themes automatically configured and ready to use.
```

Comment:
Links should be full paths to github so that they will work when viewed on NPM (relative links will not work).